cpp/tests/mtmg/threaded_test_louvain.cu

```diff
@@ -277,6 +277,7 @@ class Tests_Multithreaded
 
         std::tie(std::ignore, modularity) = cugraph::louvain<vertex_t, edge_t, weight_t, true>(
           thread_handle.raft_handle(),
+          std::nullopt,
```

Comment:
We somehow missed this in the ECG update... we don't regularly compile the MTMG code in CI.